metric-publishers/emf-metric-publisher/src/test/java/software/amazon/awssdk/metrics/publishers/emf/internal/MetricEmfConverterTest.java

```diff
@@ -178,4 +189,46 @@ void ConvertMetricCollectionToEMF_LargeCollection(){
         assertThat(emfLogs).hasSize(3);
     }
 
+    @Test
+    void ConvertMetricCollectionToEMF_dropNonNumericMetrics() {
+
+        MetricCollector metricCollector = MetricCollector.create("test");
+        metricCollector.reportMetric(CoreMetric.SERVICE_ID, "serviceId");
+        metricCollector.reportMetric(CoreMetric.OPERATION_NAME, "operationName");
+        metricCollector.reportMetric(SdkMetric.create("stringMetric", String.class, MetricLevel.INFO, MetricCategory.CORE),
+                                     "stringMetricValue");
+        metricCollector.reportMetric(HttpMetric.AVAILABLE_CONCURRENCY, 5);
+        List<String> emfLogs = metricEmfConverterDefault.convertMetricCollectionToEmf(metricCollector.collect());
+
+        assertThat(emfLogs).containsOnly("{\"_aws\":{\"Timestamp\":12345678,\"LogGroupName\":\"my_log_group_name\",\"CloudWatchMetrics\":[{\"Namespace\":"
+                                         + "\"AwsSdk/JavaSdk2\",\"Dimensions\":[[\"OperationName\",\"ServiceId\"]],\"Metrics\":[{\"Name\":\"AvailableConcurrency\"}]}]},"
+                                         + "\"AvailableConcurrency\":5,\"OperationName\":\"operationName\",\"ServiceId\":\"serviceId\"}");
+    }
+
+    @Test
+    void ConvertMetricCollectionToEMF_shouldConformToSchema() {
+        try {
+            String jsonSchema = IOUtils.toString(
+                Objects.requireNonNull(getClass().getResourceAsStream("/emfSchema.json")),
+                StandardCharsets.UTF_8
+            );
+
+            MetricCollector metricCollector = MetricCollector.create("test");
+            metricCollector.reportMetric(HttpMetric.AVAILABLE_CONCURRENCY, 5);
+            metricCollector.reportMetric(CoreMetric.SERVICE_ID, "serviceId");
```

Comment:
Can we add more metrics to simulate real-case scenarios? especially the ones with multiple values